controllers/postgres_controller.go

```diff
@@ -1211,6 +1242,12 @@ func (r *PostgresReconciler) httpPatchPatroni(log logr.Logger, ctx context.Conte
 		return err
 	}
 	defer resp.Body.Close()
+	log.V(debugLogLevel).Info("Performed request")
```

Comment:
I'm confused that the response code `resp.Status` is not checked. Shouldn't this return at least a 2xx return code? something like

~~~
if resp.StatusCode/100 != 2 {
  // log error, return error?
}
~~~